src/core/public/overlays/sidecar/sidecar_service.tsx

```diff
@@ -133,7 +134,12 @@ export class SidecarService {
         // init
         (!sidecarConfig$.value && config.dockedMode && config.paddingSize)
       ) {
-        sidecarConfig$.next({ ...sidecarConfig$.value, ...config } as ISidecarConfig);
+        const paddingSize = calculateNewPaddingSize(config.dockedMode, config.paddingSize);
```

Comment:
The config object is a partial `ISidecarConfig`, do we need to check if `config.paddingSize` be undefined here? I think we don't update padding size to MIN_SIZE for this case.